from core.shared.exceptions import NotFoundException


class CategoryNotFoundException(NotFoundException):
    pass
